content/aws_marketplace.md

```diff
@@ -16,557 +16,120 @@ product = ["client", "workstation"]
+- Fill out the registration form and [Accept the Chef License](https://docs.chef.io/chef_license_accept/).
 
-10. Run `automate-ctl restart` to restart Chef Automate
+- Select **Register** to enter Chef Automate.
 
-11. Run `chef-server-ctl restart` to restart Chef server
+![ ](/images/WelcomePage.png "Welcome Page")
 
-### Change instance size
+- Congratulations! You've started Chef Automate!
 
-To edit the Amazon Machine Images (AMI) instance size, do the following:
+![ ](/images/DashboardsPage.png "Dashboards Page")
 
-1.  Login using SSH to access the Chef Automate instance. Use the SSH
-    key pair and the IP address or public hostname that was
-    automatically assigned when the Amazon Machine Images (AMI) was
-    launched. The default user is `ec2-user`. For example:
+## Add Chef Servers
 
-    ```bash
-    ssh -i /path/to/ssh_key.pem ec2-user@<instance IP address>
-    ```
+1. Add Chef-Server Details, select the Add Chef Infra Server Button.
 
-2.  Stop the Chef Infra Server services:
+  ![ ](/images/AddChefServer.png "Add Chef Server")
 
-    ```bash
-    sudo chef-server-ctl stop
-    ```
+1. Enter the server name, FQDN, and IP address. Then select **Add Chef Infra Server** to create the server.
 
-3.  Stop then Chef Automate services:
+    - Name: Add Proper Name for the Sever.
+    - FQDN: It would be the same as Automate FQDN.
+    - IP Address: Public IP Address of the EC2-Instance.
 
-    ```bash
-    sudo automate-ctl stop
-    ```
+  <img src="/images/ChefServerDetails.png" style="width: 30%" />
 
-4.  Navigate to the Amazon Web Services (AWS) instance in the AWS
-    Management Console.
+1. The Chef Infra Server will appear in the list of servers. Select the server and view information about it.
 
-5.  From the **Actions** dropdown, select **Instance State**, and then
-    **Stop**.
+  ![ ](/images/AddChefServer01.png "Add Chef Server")
 
-6.  After the instance transitions to **Stopped**, edit the instance
-    size. From the **Actions** dropdown, select **Instance Settings**,
-    and then **Change Instance Type**.
+1. Select **Add Chef Organization**.
 
-7.  From the dropdown, select the desired instance size, and then click
-    **Apply**.
+  ![ ](/images/AddOrgPage.png "Add Org Page")
 
-8.  From the **Actions** dropdown, select **Instance State**, and then
-    click **Start**.
+1. Enter the following information:
 
-9.  After the instance has started it will have a **new public IP
-    address and public DNS**.
+    - Name: **demo**
+    - Admin User: **admin**
+    - Admin Key: _copy the key from starter kit_
 
-10. Use SSH to log into the new instance. Use the SSH key pair and new
-    IP address:
+1. Select **Add Chef Organization**.
 
-    ```bash
-    ssh -i /path/to/ssh_key.pem ec2-user@<instance IP address>
-    ```
+  <img src="/images/OrgPageDetails.png" style="width: 30%" />
 
-11. Follow the [instructions for changing the
-    hostname](#change-automate-hostname)
+## AWS Deployment Security
```

Comment:
Same problem with list item numbers as above in the **Add Chef Servers** section